src/w3c/headers.js

```diff
@@ -531,7 +534,7 @@ export function run(conf) {
   }
   if (Array.isArray(conf.wg)) {
     conf.multipleWGs = conf.wg.length > 1;
-    conf.wgHTML = joinAnd(conf.wg, (wg, idx) => {
+    conf.wgHTML = joinAndHyper(conf.wg, (wg, idx) => {
       return `the <a href='${conf.wgURI[idx]}'>${wg}</a>`;
```

Comment:
I mean, here the return value should use `hyperHTML` so that it can process the string template. (`` hyperHTML`<a>${string}</a>` `` escapes `string`, that's why we want to use it here.)